elements/pfe-navigation/demo/index.html

```diff
@@ -63,6 +63,12 @@
         cursor: pointer;
         transition: box-shadow 0.25s;
       }
+
+      pfe-icon.PFElement {
```

Comment:
Nitpick: wondering if we need `.PFElement`, if it's just for specificity I'd prefer `[class]` as a replacement, or a comment that we just need a specificity bump.

Reply:
@wesruv Yep `PFElement` was just for specificity but I reworked things but forgot to remove that extra class. I think just `pfe-icon` there would work now, But would you go over `[class]` with me? I want to make sure I understand that option for solving specificity issues.